src/themes/bankrate/theme.json

```
@@ -1214,7 +1217,10 @@
             ]
           },
           "display": "flex",
-          "flexDirection": ["column-reverse", "row-reverse", "row-reverse"]
+          "flexDirection": ["column-reverse", "row-reverse", "row-reverse"],
+          "&": {
+            "backgroundColor": "transparent"
+          }
```

do we need the "&" ?? 

```suggestion
          "backgroundColor": "transparent"
```

---

I tried without it, but it doesn't override behaviour from card.base that way

---

OK interesting, I would like to know why, but we can merge this anyway